src/order/bounded_order.lean

```diff
@@ -992,14 +992,6 @@ protected def order_bot [preorder α] [order_bot α] {P : α → Prop} (Pbot : P
 { bot := ⟨⊥, Pbot⟩,
   bot_le := λ _, bot_le }
 
-@[simp]
-lemma coe_bot  {α : Type*} [preorder α] [order_bot α] {P : α → Prop} (Pbot : P ⊥) :
-  ↑((subtype.order_bot Pbot).bot) = (⊥ : α) := rfl
```

Comment:
Why did you remove this one?

Reply:
I removed it because it was making linter angry (if I remember correctly, something about `simp` already knowing how to prove this). But I'm happy to put it back

Reply:
I put it back with a slight modification which should hopefully appease the linter